javaagent-tooling/src/test/java/io/opentelemetry/javaagent/tooling/field/GeneratedVirtualFieldNamesTest.java

```diff
@@ -39,15 +39,15 @@ void field() {
   @Test
   void setter() {
     assertEquals(
-        "set__opentelemetryVirtualField$java$lang$Runnable$java$lang$String__",
+        "__set__opentelemetryVirtualField$java$lang$Runnable$java$lang$String__",
```

Comment:
Do we need then `__` both before and after `set/get` prefix?

Reply:
it's not needed, but seems ok to me (and code is shorter to include it since it's part of the field name)